src/main/java/frc/trigon/robot/subsystems/arm/simulationarm/SimulationArmConstants.java

```diff
@@ -0,0 +1,79 @@
+package frc.trigon.robot.subsystems.arm.simulationarm;
+
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.controller.ElevatorFeedforward;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.wpilibj.simulation.ElevatorSim;
+import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
+
+public class SimulationArmConstants {
+    private static final int
+            ANGLE_MOTOR_AMOUNT = 2,
+            ELEVATOR_MOTOR_AMOUNT = 2;
+    private static final DCMotor
+            ANGLE_MOTOR_GEARBOX = DCMotor.getNEO(ANGLE_MOTOR_AMOUNT),
+            ELEVATOR_MOTOR_GEARBOX = DCMotor.getNEO(ELEVATOR_MOTOR_AMOUNT);
+    private static final double
+            ANGLE_GEAR_RATIO = 95.2,
+            ELEVATOR_GEAR_RATIO = 13.29;
+    private static final Rotation2d
+            MINIMUM_ANGLE = Rotation2d.fromDegrees(-45),
+            MAXIMUM_ANGLE = Rotation2d.fromDegrees(180);
+    private static final boolean
+            ANGLE_SIMULATE_GRAVITY = true,
+            ELEVATOR_SIMULATE_GRAVITY = true;
+    private static final double ARM_MASS = 12;
+    private static final double DRUM_RADIUS_METERS = 0.1256 / 2;
+    private static final double FULLY_OPENED_ARM_LENGTH_METERS = 1.7;
+    static final double RETRACTED_ARM_LENGTH_METERS = 0.65;
+    static final double METERS_PER_REVOLUTION = 0.1256;
```

Comment:
rename this to theoretical meters per revolution and move it to the constants since you're going to need it when trying to add the position offset to the current revolutions